tests/sentry/integrations/vsts/test_integration.py

```diff
@@ -149,7 +171,8 @@ def test_accounts_list_failure(self):
         assert resp.status_code == 200, resp.content
         assert b"No accounts found" in resp.content
 
-    def test_webhook_subscription_created_once(self):
+    @patch("sentry.integrations.vsts.VstsIntegrationProvider.get_scopes", return_value=FULL_SCOPES)
```

Comment:
@MeredithAnya I'm curious why we need this patch when the default behavior should be `FULL_SCOPES`

Reply:
@scefali I'm patching this because of the way the tests have been written, we directly call 

```python
VstsIntegrationProvider().build_integration(state)
```

but then we don't have the pipeline. This was fine when the scopes were hardcoded and it wasn't referencing the pipeline to determine which to use. I figured this was easier than rewriting all the failing tests since we do have ones tests the refresh token and basic installation